FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1509,7 +1509,8 @@ public function testPseudoSalesAreExportedCorrectly($pseudoSales, $expectedSales
 
     public function testCoverImageIsExportedAsFirstImage()
```

Comment:
Please also test the new `getPreferredImage` function.
As it is now isolated in a separate function, you can just call it with the thumbnails and image size that you want to test.